### ISSUE TITLE
Refactor code to avoid repetition and make code structured

### ISSUE THREAD

**Issue by FarbodSalamat-Zadeh** [systers/powerup-android#263]
Currently, a lot of the code in the activities are similar to each other. For example in `DressingRoomActivity`, we have this

```java
        String faceImageName = getResources().getString(R.string.face);
        faceImageName = faceImageName + getmDbHandler().getAvatarFace();
        try {
            photoNameField = ourRID.getClass().getField(faceImageName);
            faceView.setImageResource(photoNameField.getInt(ourRID));
        } catch (NoSuchFieldException | IllegalAccessException
                | IllegalArgumentException e) {
            // TODO Auto-generated catch block
            e.printStackTrace();
        }
```

This sort of structure is repeated often not only in `DressingRoomActivity` but in other activities too.

Splitting larger methods into smaller ones can also reduce repetitiveness as well as making the code more structured and readable.

**Comment by aanchal07** [comment]
Good catch. Yes you can open up an issue for this and send a PR for it. Good luck.

**Comment by chhavip** [comment]
@FarbodSalamat-Zadeh Before sending a PR, can you please illustrate how you are planning to refactor the code? A parent function in each class or a separate utils class as some functions might be repeated over different classes?

**Comment by FarbodSalamat-Zadeh** [comment]
@chhavip Sorry, I missed your comment there. 

Yes, I've added a utils class for methods that are used regularly throughout the whole app but most of the work involved moving parts of code to methods or loops. Implementing `OnClickListener` in many activities also avoided the repeated boilerplate with the inner anonymous method.

**Comment by aanchal07** [comment]
@FarbodSalamat-Zadeh Have you included the work intended for #159 too?

**Comment by FarbodSalamat-Zadeh** [comment]
@aanchal07 Yes - [on line 68 of UiUtils](https://github.com/FarbodSalamat-Zadeh/powerup-android/blob/e23996580a0ccd031c0c94d93b3119be91f005ef/PowerUp/app/src/main/java/powerup/systers/com/util/UiUtils.java#L68) there is a `setupProgressBars` method which initializes these power bars.
